resources/js/Pages/Monitors/Index.js

```diff
@@ -36,6 +36,12 @@ export default function Index(props) {
                     ))}
                 </div>
             </div>
+
+            {errors.errors && (
+                <div className="alert alert-danger">
+                    {alert(errors.errors)}
+                </div>
+            )}
```

Comment:
@rathorevaibhav I used this part to alert the user. There was a case where I input this url https://new.com.au, but when I checked on the database,, the whois tool was unable to fetch the expiration date of this domain. 
Even when I run this domain using cli, all the information was fetched but the expiration date was coming empty.
So to notify the users of such edge cases, I uses this alert.

Reply:
Sure @rathorevaibhav I can add a component that will display the the domain expiration time, in case no date is available, a message for the user.
I have created the section and I'm attaching am image for the section.